File: 1598-crawler-log-folder/1598-crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int cnt = 0;
        for(string s:logs) {
            if(s[1] == '.' && cnt != 0) cnt--;
            if(s[0] != '.') cnt++;
        }
        return cnt;
    }
};

File: 1598-crawler-log-folder/1598-crawler-log-folder.java
class Solution {
    public int minOperations(String[] logs) {
        int cnt = 0;
        for(String s:logs) {
            if(s.charAt(1) == '.' && cnt != 0) cnt--;
            if(s.charAt(0) != '.') cnt++;
        }
        return cnt;
    }
}